### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by shogof**
### Changes Made
- [x] Added new project section to portfolio.
- [x] Passed all FreecodeCamp requirments.

### Fixes
- [x] Resolved broken links in the navigation bar.
- [x] Fixed alignment issues on project cards.

### Additional Notes
- Included screenshots for reference.
- Tested on multiple browsers and devices for compatibility.
- Please review and provide feedback.

### CODE REVIEW

**Comment by shogof** [comment]
@ahzamir
Hello teacher please check my repo

File: README.md [hunk 1]
```diff
@@ -1 +1,109 @@
...
+- [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML)
+- [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS)
+
+## Installation 💻
+
+To install and set up this project, simply follow these steps:
+
+1. **Clone the repository:**
+   ```bash
+   git clone git@github.com:shogof/Technical-Documentation-page.git
```
Comment (ahzamir) [inline README.md]: - Add this project link not your technical documentation page

File: README.md [hunk 2]
```diff
@@ -1 +1,109 @@
...
+## Usage 🎯
+
+To use this project, follow these steps:
+
+1. **Installation:**
+   If you haven't already, follow the installation instructions mentioned in the [Installation](#installation-) section to clone the repository.
+
+2. **Navigate to project directory:**
+   ```bash
+   cd Technical-Documentation-page
```
Comment (ahzamir) [inline README.md]: - Same here update it to this project

File: README.md [hunk 3]
```diff
@@ -1 +1,109 @@
...
+If you'd like to contribute to this project, follow these steps:
+
+1.  **Fork the repository:**
+    Fork the repository to your own GitHub account.
+
+2.  **Clone the repository:**
+    Clone the repository to your local machine.
+
+    ```bash
+    git clone https://github.com/Shegofa/Technical-Documentation-page.git
```
Comment (ahzamir) [inline README.md]: - Same here UPDATE

**Review by ahzamir — APPROVED** [review]
### Hi @shogof 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- Although I accept and approve your project please have a look on the comments below and implement them when you have time.

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Personal Portfolio Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your repo code is clean
❌Your README.md file is professional

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: README.md (hunk 1 above)
Comment (shogof) [reply README.md]: OK Sir I do it now and push it again 
really sorry about that

File: README.md (hunk 3 above)
Comment (shogof) [reply README.md]: @ahzamir 
I did it 
Thanks in advance teacher :)

**Pull request merged**